src/helics/shared_api_library/backup/helics/helics.h

```diff
@@ -362,10 +362,7 @@ fast is the default, meaning the query travels along priority channels and takes
 existing messages; ordered means it follows normal priority patterns and will be ordered along with
 existing messages
 */
-typedef enum {
-    HELICS_SEQUENCING_MODE_FAST = 0,
-    HELICS_SEQUENCING_MODE_ORDERED = 1
-} HelicsSequencingModes;
+typedef enum { HELICS_SEQUENCING_MODE_FAST = 0, HELICS_SEQUENCING_MODE_ORDERED = 1 } HelicsSequencingModes;
```

Comment:
@phlptp is this formatting change okay or do we want to tell clang to ignore this block?

Reply:
merging the headers make the formatting different not immediately clear how we can make it so the formatting works in both cases.   I would prefer not to just ignore it but that may be the least annoying option.

Reply:
As part of generating the updated backup headers we should probably run clang-format before opening the PR with the updated header -- otherwise I think after merging this the next commit will make the header generating workflow open a PR to "update" the header.